.github/workflows/testing.yml

```
@@ -15,7 +15,12 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - run: pip install numpy
-      - run: pip install pytest
-      - run: pip install -U coverage
+      - run: pip install -U pytest coverage coveralls PyYAML
       - run: coverage run -m --source=. pytest
+      - run: coveralls --output=coverage.json
+      - run: coveralls-lcov -v -n coverage.info > coverage.json
+      - name: Coveralls GitHub Action
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Also need to specify where the lcov file is found:
```suggestion
          github-token: ${{ secrets.GITHUB_TOKEN }}
          path-to-lcov: ./coverage.lcov
```